CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+  - `layout` directory: contains the `HTML` templates.
+- [`src` directory](src): for `js` and `sass` code,
+- [`static/img` directory](static/img): for site images,
+
+
+## API
+
+The API serves (a cached version of):
+- the 3 latest blog posts tagged as `technical`, and the 3 latest blog posts tagged as `culture` for the home page,
+- all the opened positions at Lever,
+
+Handles the visitor requests to join our slack community, sending an slack invitation through [Slackin](#slackin)
+
+Its source code is under [`api` directory](api).
+
+
+## Slackin
+
+The slackin container listen for invitation requests made from the landing [api](#api).
+
+For [technical reasons](https://github.com/src-d/landing/issues/62#issuecomment-327194704), slackin container is built from the release [v0.13.1](https://github.com/rauchg/slackin/tree/0.13.1)
+
+
+# Development
+
+
+## Requirements
+
+You should already have [Go installed](https://golang.org/doc/install#install), and properly [configured the $GOPATH](https://github.com/golang/go/wiki/SettingGOPATH)
+```
+go version; # prints your go version
+echo $GOPATH; # prints your $GOPATH path
+```
+
+The project must be under the `$GOPATH`, following the Go import conventions, what means you can go to its directory running:
+```
+cd $GOPATH/src/github.com/src-d/landing
+```
+
+You need also [Yarn installed](https://yarnpkg.com/en/docs/install)
+
+```
+yarn --version; # prints your Yarn version
+```
+
+## Install and build
+
+You need to satisfy all [project requirements](#requirements), and then to run:
+
+```
+make build
+```
+
+
+## Development and running the landing locally
+
+You need to satisfy all [project requirements](#requirements), and then to run:
+
+```
+LANDING_URL=//localhost PORT=8081 make serve
+```
+It runs everything you need to get the site working at [http://localhost:8081](http://localhost:8081)
+
+Alternatively, you can start hugo, the api-server and webpack in a "three window mode", just running:
+```
+LANDING_URL=//localhost PORT=8081 yarn start
+```
+With this command, each window runs a command, that can be also ran by you in case you need to control the output of each command or in any other special case:
+* `yarn run webpack-watcher` To start webpack watcher, that will rebuild the assets when you change its sources
+* `make hugo-server` To serve the landing locally using hugo server
+* `yarn run api-run` To start the landing API at [http://localhost:8080](http://localhost:8080)
+
+### Preview the documentation site
+
+It can be seen the landing _"as it would be a documentation site"_. To do so, it is needed to run:
+```
+make develop-documentation
+```
+And then, go to [http://localhost:8081](http://localhost:8081)
+
+To rollback the changes, and see the landing as usual, just run:
+```
+make develop-documentation-destroy
+```
+
+
+## Configuration
+
+The following envars are available for API configuration
+
+- `ADDR`
```

Comment:
do you mind to add defaults here? I find it very useful to see defaults in documentation.

Reply:
I'll add a link to the place where the defaults are defined [https://github.com/src-d/landing/blob/master/api/config/config.go#L8](api/config/config.go#L8)